editor/plugins/occluder_instance_3d_editor_plugin.cpp

```diff
@@ -108,7 +108,7 @@ OccluderInstance3DEditorPlugin::OccluderInstance3DEditorPlugin() {
 	bake->set_icon(EditorNode::get_singleton()->get_editor_theme()->get_icon(SNAME("Bake"), EditorStringName(EditorIcons)));
 	bake->set_text(TTR("Bake Occluders"));
 	bake->hide();
-	bake->connect("pressed", Callable(this, "_bake"));
+	bake->connect("pressed", callable_mp(this, &OccluderInstance3DEditorPlugin::_bake));
```

Comment:
Same note here

Reply:
Since this is exactly the same as the other I'm going to mark it as resolved to keep review clean.